packages/paste-core/components/alert/__tests__/index.spec.tsx

```diff
@@ -15,75 +13,75 @@ describe('Alert', () => {
-      const wrapper: ReactWrapper = mount(
+
+      const {getByRole} = render(
         <Alert onDismiss={eventHandlerMock} variant="neutral">
           This is an alert
         </Alert>
       );
-      wrapper.find('button').simulate('click');
+
+      const button = getByRole('button');
+      fireEvent.click(button);
       expect(eventHandlerMock).toHaveBeenCalledTimes(1);
     });
   });
 
   describe('Aria roles', () => {
     it('Should add the role of status to the neutral alert', () => {
-      const wrapper: ReactWrapper = mount(<Alert variant="neutral">This is an alert</Alert>);
-      expect(wrapper.exists('[role="status"]')).toEqual(true);
+      const {getByRole} = render(<Alert variant="neutral">This is an alert</Alert>);
+      expect(getByRole('status')).toBeDefined();
     });
 
     it('Should add the role of alert to the error alert', () => {
-      const wrapper: ReactWrapper = mount(<Alert variant="error">This is an alert</Alert>);
-      expect(wrapper.exists('[role="alert"]')).toEqual(true);
+      const {getByRole} = render(<Alert variant="error">This is an alert</Alert>);
+      expect(getByRole('alert')).toBeDefined();
     });
 
     it('Should add the role of alert to the warning alert', () => {
-      const wrapper: ReactWrapper = mount(<Alert variant="warning">This is an alert</Alert>);
-      expect(wrapper.exists('[role="alert"]')).toEqual(true);
+      const {getByRole} = render(<Alert variant="warning">This is an alert</Alert>);
+      expect(getByRole('alert')).toBeDefined();
     });
 
     it('Should add the provided role to the alert', () => {
-      const wrapper: ReactWrapper = mount(
+      const {getByRole} = render(
         <Alert role="tab" variant="error">
           This is an alert
         </Alert>
       );
-      expect(wrapper.exists('[role="tab"]')).toEqual(true);
+      expect(getByRole('tab')).toBeDefined();
     });
   });
 
   describe('Variant neutral', () => {
     it('Should render a neutral alert', (): void => {
-      const tree = renderer
-        .create(
-          <Theme.Provider theme="console">
-            <Alert variant="neutral">This is an alert</Alert>
-          </Theme.Provider>
-        )
-        .toJSON();
-      expect(tree).toMatchSnapshot();
+      const {asFragment} = render(
+        <Theme.Provider theme="console">
+          <Alert variant="neutral">This is an alert</Alert>
+        </Theme.Provider>
+      );
+      expect(asFragment()).toMatchSnapshot();
     });
 
     it('Should render a neutral alert with dismiss button', (): void => {
-      const tree = renderer
-        .create(
-          <Theme.Provider theme="console">
-            <Alert onDismiss={onDismissMock} variant="neutral">
-              This is an alert
-            </Alert>
-          </Theme.Provider>
-        )
-        .toJSON();
-      expect(tree).toMatchSnapshot();
+      const {asFragment} = render(
+        <Theme.Provider theme="console">
```

Comment:
NIT: We recently started using a testing theme, essentially mocking our theme so it is constant throughout time, and token changes do not require test updates.

To use it you would replace the `ThemeProvider` wrapper with:
```
<CustomizationProvider theme={TestTheme}>
    {/* the component being tested */}
</CustomizationProvider>
```